index.js

```diff
@@ -50,7 +50,7 @@ const camundaCloud82Rules = withConfig({
 }, { version: '8.2' });
 
 const camundaCloud83Rules = withConfig({
-  ...camundaCloud82Rules,
+  ...withoutKeys(camundaCloud82Rules, [ 'inclusive-gateway' ]),
```

Comment:
```suggestion
  ......omit(camundaCloud82Rules, 'inclusive-gateway'),
```

We have a pattern for this already.

Reply:
Right 🍡